api-docs/cloud-serverless/ref.yml

```
@@ -14680,8 +14680,8 @@ tags:
       | &nbsp;Code&nbsp; | Status                   | Description      |
       |:-----------:|:------------------------ |:--------------------- |
       | `200`       | Success                  |                       |
-      | `201`       | Created                           | Successfully created a resource. The response body may contain details. |
-      | `204`       | No content      | The request succeeded, but InfluxDB doesn't typically return a response body for the operation. |
+      | `201`       | Created                  | Successfully created a resource. The response body may contain details. |
+      | `204`       | No content               | The request succeeded. For _writes_, a response body is returned if part of the request failed. See the write API spec for details. |
```

```suggestion
      | `204`       | No content               | The request succeeded. For _writes_ that have [rejected points](/influxdb/cloud-serverless/write-data/troubleshoot/#troubleshoot-rejected-points), the [`/write` and `/api/v2/write` endpoints](#tag/Write) return a response body with error details. |
```

Thanks for this. I chose to remove the blurb about writes, and just leave it more generic (maybe a bit of self-serving DRYness), but we can add it back.